docs/src/physics/turbulence_closures.md

```diff
@@ -38,15 +38,16 @@ and diffusive flux divergence
 \boldsymbol{\nabla} \boldsymbol{\cdot} \boldsymbol{q}_c = - \left [ \kappa_{h} \left ( \partial_x^2 + \partial_y^2 \right )
                                     + \kappa_{v} \partial_z^2 \right ] c \, ,
 ```
-in terms of the horizontal viscosities and diffusivities ``\nu_h`` and ``\kappa_{h}`` and the
-vertical viscosity and diffusivities ``\nu_v`` and ``\kappa_{v}``.
-Each tracer may have a unique diffusivity components ``\kappa_h`` and ``\kappa_v``.
+in terms of the horizontal viscosities and diffusivities, ``\nu_h`` and ``\kappa_h``, and the
+vertical viscosity and diffusivities, ``\nu_z`` and ``\kappa_z``. Each tracer may have a unique
+diffusivity components ``\kappa_h`` and ``\kappa_v``.
 
-## Constant anisotropic biharmonic diffusivity
+## Scalar biharmonic diffusivity
+
+A constant biharmonic diffusivity implies a constant tensor diffusivity ``\nu_{j k}`` and
+stress``\boldsymbol{\tau}_{ij} = \nu_{j k} \partial_k^3 u_i`` with non-zero components
+``\nu_{11} = \nu_{22} = \nu_h`` and ``\nu_{33} = \nu_z``.
 
-In Oceananigans.jl, a constant anisotropic biharmonic diffusivity implies a constant tensor
-diffusivity ``\nu_{j k}`` and stress ``\boldsymbol{\tau}_{ij} = \nu_{j k} \partial_k^3 u_i`` with non-zero
-components ``\nu_{11} = \nu_{22} = \nu_h`` and ``\nu_{33} = \nu_v``.
 With this form the kinematic stress divergence becomes
 ```math
 \boldsymbol{\nabla} \boldsymbol{\cdot} \boldsymbol{\tau} = - \left [ \nu_h \left ( \partial_x^2 + \partial_y^2 \right )^2
```

Comment:
Here we could add that in case of an Isotropic (or `ThreeDimensional`) scalar biharmonic diffusivity the formulation is now
```
ν ∇ ∇u
```

also biharmonic has opposite sign than normal harmonic

Reply:
Also theres nothing called "anisotropic biharmonic diffusivity" right?

Reply:
Yeah, `Biharmonic` now follows the same rules as the harmonic diffusivity, i.e., `ThreeDimensionalFormulation`, `HorizontalFormulation` and `VerticalFormulation` where the fluxes for full 3D are gradients of laplacians, while for horizontal horizontal gradient of horizontal laplacians and vertical only the third order derivative in z

Reply:
Damn! I merged and then saw the remarks!